EIPS/eip-2930.md

```diff
@@ -29,6 +29,15 @@ This EIP serves two functions:
 
 ## Specification
 
+### Definitions
+
+**`TransactionType`** `1`.  See [EIP-2718](./eip-2718.md)
+
+**`ChainId`** The transaction is valid if this value is `0` or it is included in a block on a chain whose ID is equal to this value.
```

Comment:
Now you changed it back so `0` means wildcard. Why?

Reply:
Good catch, copy/paste error (copied from 2711, where `0` cross-chain transactions are allowed).  Fixed now.